### ISSUE TITLE
V-Diffusion from Katherine

### ISSUE THREAD

**Issue by patrickvonplaten** [huggingface/diffusers#322]
### Model/Pipeline/Scheduler description

Katherine's trained models in v-diffusion are very powerful especially if used with CLIP-guidance. 
Would be great to have them included here as well! 

https://github.com/crowsonkb/v-diffusion-pytorch

### Open source status

- [X] The model implementation is available
- [X] The model weights are available (Only relevant if addition is not a scheduler).

### Provide useful links for the implementation

Happy to help here if anybody is interested :-)

**Comment by apolinario** [comment]
I would suggest that additionally to the models pre-trained by Katherine that are on the original repo, that when models are adapted to diffusers, to also add [nshepperd's original `guided-diffusion` 512x512 unconditional model fine-tuned on openimages](https://set.zlkj.in/models/diffusion/512x512_diffusion_uncond_openimages_epoch28_withfilter.pt) which produces very good image quality when guided

**Comment by gradjitta** [comment]
Hi @patrickvonplaten I can take this up and also any pointers to help me get started are most appreciated.

**Comment by patrickvonplaten** [comment]
That sounds great @gradjitta!

To begin with I think you should try to get inference working with the original codebase (ideally in a google colab) and then we can start porting the code over to `diffusers`. Would you like to setup a google colab we could work on together maybe? :-)

**Comment by gradjitta** [comment]
@patrickvonplaten That sounds good. Would it be OK if I started working on it 15th of this month?

**Comment by patrickvonplaten** [comment]
Sure!
